docs/ref/middleware.txt

```diff
@@ -530,6 +545,9 @@ decorator are exempt from this requirement.
 
 **Methods and Attributes**
 
+Subclass the middleware and override these to customize behavior for
+unauthenticated requests.
```

Comment:
Without this sentence, it was unclear why these methods and attributes are documented.